charts/vector/templates/daemonset.yaml

```diff
@@ -12,7 +12,7 @@ spec:
     matchLabels:
       {{- include "vector.selectorLabels" . | nindent 6 }}
   {{- with .Values.updateStrategy }}
```

Comment:
```suggestion
   minReadySeconds: {{ $.Values.minReadySeconds }}
   {{- with .Values.updateStrategy }}
```

@rbakhtaraev would you mind swapping these lines in this proposal?

Reply:
If moved outside of the `with` block, the `$` should be dropped as well, I beleive.

Reply:
https://github.com/vectordotdev/helm-charts/pull/308 This PR already has the correct changes.

Reply:
This is true. @spencergilbert how do you feel about reviewing the other PR instead? Or would you prefer moving forward on this proposal?

Reply:
yep, fixed it

Reply:
awesome, thanks.